dftimewolf/lib/processors/turbinia_artifact.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+"""Processes a directory of artifacts with Turbinia."""
+
+from __future__ import unicode_literals
+from __future__ import absolute_import
+from __future__ import print_function
+
+import os
+
+from turbinia import evidence
+from turbinia import TurbiniaException
+
+from dftimewolf.lib.containers import containers
+from dftimewolf.lib.modules import manager as modules_manager
+from dftimewolf.lib.processors.turbinia_gcp import TurbiniaProcessorBase
+
+# pylint: disable=no-member
+
+class TurbiniaArtifactProcessor(TurbiniaProcessorBase):
+  """Processes Google Cloud (GCP) disks with Turbinia.
```

Comment:
This processes local artifacts / files, rather than GCP disks, correct?

Reply:
Yes!  Thanks.  Fixed.